model_compiler/tests/model_compiler/compilers/test_torch_model_file_to_onnx_model.py

```diff
@@ -18,27 +19,27 @@ class ConfigTestCase(TestCase):
     def test_from_json(self):
         self.assertEqual(Config.from_json({'input_names': ['input'],
                                            'input_shapes': [1, 2, 3, 4],
-                                           'data_type': 'FLOAT16',
+                                           'data_type': torch.float,     # pylint: disable=no-member
```

Comment:
I think json config and env are common configs, it should not be related to frameworks like torch, tf etc.